app/routes/games/helpers.js

```diff
@@ -89,6 +148,13 @@ function renderPage(req, res, template, populate=null)
 						: Number(req.query.page)
 					: 1;
 
+			// iterate game releases to add file sizes
+			for (let k = 0; k < game.releases.length; k++) {
+				const commitId = game.releases[k].commitId;
+				const humanFileSize = await releaseSize(game.location, game.slug, commitId)
```

Comment:
So is this iterating and calling for the size on every page load? Has any consideration been given to trying to cache/store the values so we don't have to get them every time? Or is this ultimately fast enough to not be a bother?

Reply:
You are correct to raise these questions. 

Gonna re-work the release model to populate `releaseCompressedSize` (if a value has not been passed) when the object is saved.